src/main/resources/static/admin/AdminPanelPage.html

```diff
@@ -0,0 +1,369 @@
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Lorem Ipsum Dolor Sit Amet</td>
+                        <td>Mark Corrigan</td>
+                        <td>25th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Ipsum Lorem Dolor Sit Amet</td>
+                        <td>Jeremy Usbourne</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Sit Amet Dolor Ipsum</td>
+                        <td>Super Hans</td>
+                        <td>20th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Alan Johnson</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
+                                                                                                src="adminimg/icn_trash.png"
+                                                                                                title="Trash" alt="">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><input type="checkbox"></td>
+                        <td>Dolor Lorem Amet</td>
+                        <td>Dobby</td>
+                        <td>26th May 2017</td>
+                        <td><input type="image" src="adminimg/icn_edit.png" title="Edit"><input type="image"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)